.gitignore

```diff
@@ -351,4 +351,5 @@ MigrationBackup/
 
 test/ffmpeg-20200807-fab00b0-win64-dev/*
 test/ffmpeg-20200807-fab00b0-win64-shared/*
-*.bmp
\ No newline at end of file
+*.bmp
+/src/SIPSorceryMedia.FFmpeg.sln
```

Comment:
This solution file is needed. My only concern was if the SIPSorcery package was being referenced from the SIPSorceryMedia.FFmpeg project. Can you add it back in? I can sort out the references.